app/experimenter/experiments/api/v5/queries.py

```diff
@@ -53,7 +54,19 @@ def resolve_outcomes(root, info):
         return Outcomes.all()
 
     def resolve_targeting_config_slug(root, info):
-        return root._text_choices_to_label_value_list(NimbusExperiment.TargetingConfig)
+        return [
+            NimbusExperimentTargetingConfigSlugChoice(
+                label=member.label,
+                value=choice_name,
+                application_values=NimbusExperiment.TARGETING_CONFIGS[
```

Comment:
Maybe `application_names` is more appropriate?

Reply:
I went with `application_values` here since it's essentially the same kind of data as `value` - i.e. choice names. It's all kind of a tangle of terms.